modules/reference/pages/rpk/rpk-x-options.adoc

```diff
@@ -42,6 +42,14 @@ You can use this if you have well known certificates set up on your Kafka API. I
 
 '''
 
+=== tls.insecure_skip_verify
+
+A boolean that disables `rpk` from verifying the broker's certificate chain.
+
+*Example*: `tls.insecure_skip_verify=true`
+
+'''
+
 === tls.ca
 
 A filepath to a PEM encoded CA certificate file to talk to your broker's Kafka API listeners with mTLS.
```

Comment:
```suggestion
A filepath to a PEM-encoded CA certificate file to talk to your broker's Kafka API listeners with mTLS.
```
@r-vasquez just a little thing, but you could add this hypen to the other property descriptions just to be consistent with these new ones.

Reply:
Done, added in https://github.com/redpanda-data/docs/pull/1009/commits/220a78b510934234ebc740cd4349b0cb85fa5bd7.

Thanks!

Reply:
thank you!